Package.swift

```diff
@@ -1,5 +1,13 @@
 import PackageDescription
 
 let package = Package(
-  name: "Quick"
+    name: "Quick",
+    targets: [
+        Target(name: "QuickTests", dependencies: [.Target(name: "Quick"), .Target(name: "QuickTestHelpers")]),
+        Target(name: "QuickFocusedTests", dependencies: [.Target(name: "Quick"), .Target(name: "QuickTestHelpers")]),
+        Target(name: "QuickTestHelpers", dependencies: [.Target(name: "Quick")]),
+    ],
+    dependencies: [ // TODO: This ought to be `testDependencies:` but there seems to be a swiftpm bug causing the lib to not get linked then
```

Comment:
Could you include a Swift JIRA issue number or a link to a swift-package-manager pull request here? That way a future contributor could determine whether the bug has been fixed and this code could be changed.